src/Drivers/check_determinant.cpp

```diff
@@ -178,7 +180,8 @@ int main(int argc, char **argv)
           PosType dr   = sqrttau * delta[iel];
           bool isValid = els.makeMoveAndCheck(iel, dr);
 
-          if (!isValid) continue;
+          if (!isValid)
+            continue;
```

Comment:
One line is better.

Reply:
So that's a different setting but I agree.
```
AllowShortIfStatementsOnASingleLine: true
```